src/extension-runners/chromium.js

```diff
@@ -273,7 +273,7 @@ export class ChromiumExtensionRunner {
 
     log.debug(`Creating reload-manager-extension in ${extPath}`);
 
-    await asyncMkdirp(extPath);
+    await fs.mkdir(extPath, { recursive: true });
```

Comment:
Added to Node 10 (as promised)

https://nodejs.org/api/fs.html#fspromisesmkdirpath-options